trio/_core/_run.py

```diff
@@ -889,18 +898,18 @@ async def __aexit__(self, etype, exc, tb):
                 # see test_simple_cancel_scope_usage_doesnt_create_cyclic_garbage
                 del _, combined_error_from_nursery, value, new_exc
 
-    def __enter__(self):
+    def __enter__(self) -> None:
         raise RuntimeError(
             "use 'async with open_nursery(...)', not 'with open_nursery(...)'"
         )
 
-    def __exit__(self):  # pragma: no cover
+    def __exit__(self) -> None:  # pragma: no cover
         assert False, """Never called, but should be defined"""
 
 
 def open_nursery(
     strict_exception_groups: bool | None = None,
-) -> AbstractAsyncContextManager[Nursery]:
+) -> NurseryManager:
     """Returns an async context manager which must be used to create a
     new `Nursery`.
```

Comment:
Not entirely sure about this one.

Reply:
nurserymanager is an internal class. it should just be asynccontextmanager.